smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/smithy-lock-test.json

```diff
@@ -0,0 +1,11 @@
+{
+  "_AUTOGENERATED_FILE_DO_NOT_MODIFY_MANUALLY_":"",
+  "version":"1.0",
+  "config_hash":-1856284556,
+  "repositories":["https://repo.maven.apache.org/maven2"],
+  "artifacts":{
+    "software.amazon.smithy:smithy-aws-traits:1.37.0":{
+      "shaSum":"723b00f4473c0f3f308b57b2c95b32fad669904c68d8e287606304d841cd52ff"
```

Comment:
Can you say what kind of sha it is? e.g. "sha256": "..."

Reply:
Will do.